contrib/graph-gophers/graphql-go/graphql_test.go

```diff
@@ -210,3 +186,40 @@ func TestAnalyticsSettings(t *testing.T) {
 		assertRate(t, mt, 0.23, WithAnalyticsRate(0.23))
 	})
 }
+
+func TestNamingSchema(t *testing.T) {
+	genSpans := namingschematest.GenSpansFn(func(t *testing.T, serviceOverride string) []mocktracer.Span {
```

Comment:
So why again is `serviceOverride` a string, and not a bool?

Reply:
I think it could be a bool but it would require changes in many other packages. I'll deal with this issue in the refactor PR I'm preparing after all these are merged 😄